tests/test_check_parallel.py

```diff
@@ -48,109 +73,158 @@ def helperCheckParallelVerboseMismatch(self, test_name, hpss_path, zstash_path=Z
-        print('Messing up the MD5 of all of the files with an even id.')
-        cmd = ['sqlite3', '{}/index.db'.format(self.cache), 'UPDATE files SET md5 = 0 WHERE id % 2 = 0;']
+        shutil.copy(
+            "{}/index.db".format(self.cache), "{}/index_old.db".format(self.cache)
+        )
+        print("Messing up the MD5 of all of the files with an even id.")
+        cmd = [
+            "sqlite3",
+            "{}/index.db".format(self.cache),
+            "UPDATE files SET md5 = 0 WHERE id % 2 = 0;",
+        ]
         run_cmd(cmd)
-        cmd = '{}zstash check -v --hpss={} --workers=3'.format(zstash_path, self.hpss_path)
+        # FIXME: Incompatible types in assignment (expression has type "str", variable has type "List[str]") mypy(error)
+        cmd = "{}zstash check -v --hpss={} --workers=3".format(  # type: ignore
+            zstash_path, self.hpss_path
+        )
         output, err = run_cmd(cmd)
         # These files have an even `id` in the sqlite3 table.
-        expected_present = ['md5 mismatch for: dir/file1.txt', 'md5 mismatch for: file3.txt',
-                            'ERROR: 000001.tar', 'ERROR: 000004.tar', 'ERROR: 000002.tar']
+        expected_present = [
+            "md5 mismatch for: dir/file1.txt",
+            "md5 mismatch for: file3.txt",
+            "ERROR: 000001.tar",
+            "ERROR: 000004.tar",
+            "ERROR: 000002.tar",
+        ]
         # These files have an odd `id` in the sqlite3 table.
-        expected_absent = ['ERROR: 000000.tar', 'ERROR: 000003.tar', 'ERROR: 000005.tar']
+        expected_absent = [
+            "ERROR: 000000.tar",
+            "ERROR: 000003.tar",
+            "ERROR: 000005.tar",
+        ]
         self.check_strings(cmd, output + err, expected_present, expected_absent)
         # Put the original index.db back.
-        os.remove('{}/index.db'.format(self.cache))
-        shutil.copy('{}/index_old.db'.format(self.cache), '{}/index.db'.format(self.cache))
+        os.remove("{}/index.db".format(self.cache))
+        shutil.copy(
+            "{}/index_old.db".format(self.cache), "{}/index.db".format(self.cache)
+        )
         os.chdir(TOP_LEVEL)
 
-        print('Verifying the data from database with the actual files')
+        print("Verifying the data from database with the actual files")
         # Checksums from HPSS
-        cmd = ['sqlite3', '{}/{}/index.db'.format(self.test_dir, self.cache), 'SELECT md5, name FROM files;']
+        cmd = [
+            "sqlite3",
+            "{}/{}/index.db".format(self.test_dir, self.cache),
+            "SELECT md5, name FROM files;",
+        ]
         output_hpss, err_hpss = run_cmd(cmd)
         hpss_dict = {}
 
-        for l in output_hpss.split('\n'):
-            l = l.split('|')
-            if len(l) >= 2:
-                f_name = l[1]
-                f_hash = l[0]
+        for checksum in output_hpss.split("\n"):
```

Comment:
Fixes `flake8` code E741

>Variables named I, O, and l can be very hard to read. This is because the letter I and the letter l are easily confused, and the letter O and the number 0 can be easily confused.
Change the names of these variables to something more descriptive.
https://www.flake8rules.com/rules/E741.html